test/test_signatures_command.rb

```diff
@@ -0,0 +1,162 @@
+require 'helper'
+require "rubygems/commands/signatures_command"
+
+class TestSignaturesCommand < Gem::TestCase
+  include SigstoreAuthHelper
+  include FulcioHelper
+  include RekorHelper
+
+  def setup
+    super
+
+    @gem_path = gem_path("hello-world.gem")
+    @cmd = Gem::Commands::SignaturesCommand.new
+  end
```

Comment:
I chose to move the request stubbing into the tests. There is no overlap between signing and verifying requests.